addons/mail/models/mail_message.py

```diff
@@ -15,6 +15,8 @@
 _logger = logging.getLogger(__name__)
 _image_dataurl = re.compile(r'(data:image/[a-z]+?);base64,([a-z0-9+/\n]{3,}=*)\n*([\'"])(?: data-filename="([^"]*)")?', re.I)
 
+IF_AUTHOR = object()
```

Comment:
What is that ?

Reply:
See https://github.com/odoo/odoo/pull/82912#discussion_r786971594 I suggested an enum (like the `odoo.fields.Command`) but seb preferred a placeholder object. My goal was mostly to avoid the weird magic string thing since for clarity they preferred avoiding a boolen

Reply:
I feel the extra syntax needed for Enum is overkill for this very specific case, it's not really worth having extra stuff at the top of the file.

Do we absolutely want to avoid the hardcoded string? I feel like it was the best compromise in this case.